src/Robot.elm

```diff
@@ -7,17 +7,14 @@ import Html.Attributes as Attributes exposing (style)
 import Html.Events exposing (onClick)
 import Json.Decode
 
-main =
-    Browser.element { init = init, update = update, view = view, subscriptions = subscriptions }

```

Comment:
Don't remove this `main`, it's useful to be able to run submodules independently.